src/main/java/com/sun/wineshop/domain/dto/response/CategoryResponseDTO.java

```diff
@@ -0,0 +1,10 @@
+package com.sun.wineshop.domain.dto.response;
+
+import lombok.Data;
+
+@Data
+public class CategoryResponseDTO {
```

Comment:
```suggestion
public record CategoryResponse {
```
nên bỏ DTO đi cho gọn, vì nó nằm trong dto pakage rồi mà.